tools/myst/enc/enc.c

```diff
@@ -871,9 +952,11 @@ int myst_tcall_get_cpuinfo(char* buf, size_t size)
     return retval;
 }
 
-OE_SET_ENCLAVE_SGX(
+OE_SET_ENCLAVE_SGX_KSS(
```

Comment:
I'd prefer using `OE_SET_ENCLAVE_SGX_2`. But it depends on https://github.com/openenclave/openenclave/pull/4068.